src/main/java/frc/robot/commands/ShooterTestCommand.java

```diff
@@ -95,19 +100,36 @@ public void initialize() {
   public void execute() {
     if ((int)input_top_motor.getNumber(0) > 5500) {
       input_top_motor.setNumber(5500);
+       inputTopVelocity = (int)input_top_motor.getNumber(0); // This should just set it to 5500
     } else {
-      input_top_motor.getNumber(0);
+      inputTopVelocity = (int)input_top_motor.getNumber(0);
     }
     if ((int)input_bottom_motor.getNumber(0) > 5500) {
       input_bottom_motor.setNumber(5500);
     } else {
       input_bottom_motor.getNumber(0);
     }
+
+    inputTopVelocity = (int)input_top_motor.getNumber(0);
+    inputBottomVelocity = (int)input_bottom_motor.getNumber(0);
+
+    top_motor_velocity.setNumber(shooter.getVelocity(MotorPlacement.TOP));
+    bottom_motor_velocity.setNumber(shooter.getVelocity(MotorPlacement.BOTTOM));
+
+    shooter.set(inputTopVelocity, inputBottomVelocity);
   }
 
   // Called once the command ends or is interrupted.
   @Override
   public void end(boolean interrupted) {
+    top_motor_velocity.setNumber(0);
+    bottom_motor_velocity.setNumber(0);
+    top_motor_position.setNumber(0);
+    bottom_motor_velocity.setNumber(0);
+    top_motor_current.setNumber(0);
+    bottom_motor_current.setNumber(0);
+    input_top_motor.setNumber(0);
+    input_bottom_motor.setNumber(0);
   }
```

Comment:
I don't want you to set all the shuffleboard fields to zero, I want you to stop all the motors.